code/__DEFINES/role_preferences.dm

```diff
@@ -71,7 +71,7 @@ GLOBAL_LIST_INIT(special_roles, list(
 	ROLE_DEVIL = /datum/game_mode/devil,
 	ROLE_OVERTHROW = /datum/game_mode/overthrow,
 	ROLE_HIVE = /datum/game_mode/hivemind,
-	ROLE_INTERNAL_AFFAIRS = /datum/game_mode/traitor/internal_affairs,
+	//ROLE_INTERNAL_AFFAIRS = /datum/game_mode/traitor/internal_affairs - This gamemode is no longer in rotation anyway. Commented out to prevent confusion between the IAA role and the antag
```

Comment:
Undocumented change
also out of scope of the pr

Reply:
Reverted